portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,190 @@
+@import url('https://fonts.googleapis.com/css?family=Major Mono Display');
+@import url('https://fonts.googleapis.com/css?family=Codystar');
+@import url('https://fonts.googleapis.com/css?family=Nothing You Could Do');
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  margin-left: 15%;
+  margin-right: 15%;
 }
 
 #greeting-container {
   margin-top: 20px;
+  
+}
+
+#TechCorner {
+    background-color: #F0EDE5;
+    text-align: right;
+}
+
+#Contact {
+    background-color: #F0EDE5;
+    text-align: right;
+}
+
+
+img.image {
+    float: right;
+    padding: 20px;
+}
+
+p.intro, div a{
+    width: 50%;
+    white-space: pre-wrap;  
+    word-wrap: break-word;
+
+}
+
+/* travel log slideshow setting */
+.slideshow-container{
+    box-sizing: border-box;
+    width: 50%;
+    position: relative;
+    margin: auto;
+}
+
+
+#firstSlide {
+    box-sizing: border-box;
+    display: block;
+}
+
+.mySlides {
+    box-sizing: border-box;
+    display: none;
+}
+
+
+/* Next & previous buttons */
+.prev, .next {
+  cursor: pointer;
+  position: absolute;
+  top: 50%;
+  width: auto;
+  margin-top: -22px;
+  padding: 16px;
+  color: white;
+  font-weight: bold;
+  font-size: 18px;
+  transition: 0.6s ease;
+  border-radius: 0 3px 3px 0;
+  user-select: none;
+}
+
+/* Position the "next button" to the right */
+.next {
+  right: 0;
+  border-radius: 3px 0 0 3px;
+}
+
+/* On hover, add a black background color with a little bit see-through */
+.prev:hover, .next:hover {
+  background-color: rgba(0,0,0,0.8);
+}
+
+/* Caption text */
+.text {
+  color: #EAE6DA;
+  font-size: 20px;
+  font-family: "Lato";
+  padding: 8px 12px;
+  position: absolute;
+  bottom: 8px;
+  width: 100%;
+  text-align: center;
+  text-shadow: 2px 2px #2E4A62;
+}
+
+/* Number text (1/3 etc) */
+.numbertext {
+  color: #EAE6DA;
+  font-size: 18px;
+  font-family: "Lato";
```

Comment:
I would suggest adding fallback fonts for all of these. https://css-tricks.com/css-basics-fallback-font-stacks-robust-web-typography/

Reply:
done